tests/Console/Command/DescribeCommandTest.php

```diff
@@ -73,11 +73,11 @@ public function testExecuteOutputWithDecoration()
 Fixes stuff.
 Replaces bad stuff with good stuff.
 
-Fixer is configurable using following options:
+Fixer is configurable using following option:
```

Comment:
yeah, prop. together with this line https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2876/files#diff-88cad9ab17d8e3b415e648e3fd31f3d2L180